### PR TITLE
refactor: Refactor identifier results in the platform adapter.

### PR DESCRIPTION
**Pull request by tajespasarela**
EX-6303

### CODE REVIEW

File: packages/x-adapter-platform/src/mappers/requests/__tests__/identifier-results-request.mapper.spec.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    expect(identifierResultsRequestMapper(internalRequest, {})).toStrictEqual({
+      query: 'chips',
+      rows: 2,
+      start: 14,
+      origin: 'url:external',
+      instance: 'empathy',
+      env: 'test',
+      lang: 'en',
+      device: 'mobile',
+      scope: 'mobile'
```
Comment (mnavarroespinosa) [inline packages/x-adapter-platform/src/mappers/requests/__tests__/identifier-results-request.mapper.spec.ts:29]:
```suggestion
      extraParams: {
        instance: 'empathy',
        env: 'test',
        lang: 'en',
        device: 'mobile',
        scope: 'mobile'
      }
```

**Review by mnavarroespinosa — COMMENTED** [review]
- Could you rename the file `sku-search-response.mapper.spec.ts` to `identifier-results-response.mapper.spec.ts`?
- I think these other two files can be deleted: `sku-search.response.ts` and `platform-sku-search.response.ts`

**Review by mnavarroespinosa — COMMENTED** [review]
There is a build error due to a wrong import in `packages/x-adapter-platform/src/types/response.types.ts`. 

The model `PlatformResult` should be imported from:
`import { PlatformResult } from './models/result.model';`

File: packages/x-adapter-platform/src/schemas/models/result.schema.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { createMutableSchema, Schema } from '@empathyco/x-adapter-next';
+import { Result } from '@empathyco/x-types';
+import { getTaggingInfoFromUrl } from '@empathyco/x-utils';
+import { PlatformResult } from '../../types/models/result.model';
+
+export const resultSchema = createMutableSchema<Schema<PlatformResult, Result>>({
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply packages/x-adapter-platform/src/schemas/models/result.schema.ts:6]: yes, lets avoid the `mutable`, all of the will be mutable out of the box. So I will rename the others to `resultSchema`

File: packages/x-adapter-platform/src/mappers/requests/__tests__/identifier-results-request.mapper.spec.ts (hunk 1 above)
Comment (tajespasarela) [reply packages/x-adapter-platform/src/mappers/requests/__tests__/identifier-results-request.mapper.spec.ts]: done

File: packages/x-adapter-platform/src/mappers/__tests__/response/sku-search-response.mapper.spec.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import { skuSearchResponseMapper } from '../../response/sku-search-response.mapper';
+// eslint-disable-next-line max-len
+import { identifierResultsResponseMapper } from '../../responses/identifier-results-response.mapper';
 // eslint-disable-next-line max-len
 import { platformSkuSearchResponse } from '../../../__tests__/__fixtures__/platform-sku-search.response';
 import { skuSearchResponse } from '../../../__tests__/__fixtures__/sku-search.response';
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply packages/x-adapter-platform/src/mappers/__tests__/response/sku-search-response.mapper.spec.ts]: renamed

File: packages/x-adapter-platform/src/mappers/__tests__/response/sku-search-response.mapper.spec.ts [hunk 4]
```diff
@@ -7,7 +8,10 @@ import { PlatformSkuSearchResponse } from '../../../types/response.types';
 describe('sku search response mapper tests', () => {
   it('should map the sku search response', () => {
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply packages/x-adapter-platform/src/mappers/__tests__/response/sku-search-response.mapper.spec.ts]: done

**Review by mnavarroespinosa — DISMISSED** [review]
Looks good to me!

**Review by mnavarroespinosa — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Pull request merged**